pkg/ring/kv/memberlist/memberlist_client_test.go

```diff
@@ -144,6 +144,14 @@ func (d dataCodec) Decode(b []byte) (interface{}, error) {
 	return out, err
 }
 
+func (d dataCodec) DecodeMultiKey(map[string][]byte) (interface{}, error) {
+	return nil, errors.New("memberlist does not support DecodeMultiKey")
+}
+
+func (d dataCodec) EncodeMultiKey(interface{}) (map[string][]byte, error) {
+	return nil, errors.New("memberlist does not support EncodeMultiKey")
```

Comment:
Should we say `memberlist does not support...` or `dataCodec does not support...`?

Reply:
Agree, makes more sense.